src/components/ppom/sharedConstants.js

```diff
@@ -0,0 +1,39 @@
+import { maliciousAddress } from '../../sample-addresses';
```

Comment:
moved the malicious tx's in this file, so they can be used both in the ppom tx and in the ppom batch txs